pkg/console/operator/status.go

```diff
@@ -0,0 +1,257 @@
+package operator
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	operatorsv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
```

Comment:
Detailing out the purpose behind the status/conditions here to ensure we get it right.